superset/db_engine_specs/databricks.py

```diff
@@ -37,7 +51,7 @@ class DatabricksODBCEngineSpec(BaseEngineSpec):
     # the syntax for the ODBC engine is identical to the Hive one, so
     # we can reuse the expressions from `HiveEngineSpec`
     # pylint: disable=protected-access
-    _time_grain_expressions = HiveEngineSpec._time_grain_expressions
+    _time_grain_expressions = DatabricksHiveEngineSpec._time_grain_expressions
```

Comment:
You can also define `time_grain_expressions` outside the classes, and then reuse it in both classes, I think it might be clearer:

```python
time_grain_expression = { ... }

class DatabricksHiveEngineSpec(HiveEngineSpec):
    _time_grain_expressions = time_grain_expressions
    ...

class DatabricksODBCEngineSpec(BaseEngineSpec):
    _time_grain_expressions = time_grain_expressions
    ...
```

Reply:
Sounds good here! :D